BlueprintUI/Sources/Element/UIViewElement.swift

```diff
@@ -154,6 +154,13 @@ private final class UIViewElementMeasurer {
 
         let view = measurementView(for: element)
 
+        /// Ensure that during measurement / sizing, the inherited `Environment` is available
+        /// to any child `BlueprintView`s. We must manually wire this property up, as the
+        /// measurement views are not in the view hierarchy.
+
+        view.nativeViewNodeBlueprintEnvironment = environment
+        defer { view.nativeViewNodeBlueprintEnvironment = nil }
```

Comment:
Not sure if this matters or is necessary, but should we store the previous value and then restore it in `defer` rather than assuming it's `nil`?

Reply:
Blueprint is the only thing that sets this property or reads it; so I think nil-ing out is correct / better; also to avoid retaining anything stored in the environment in the static prototype views.